### DISCUSSION TITLE
Refreshing Revolt for Web - Revolt

### DISCUSSION THREAD

**Discussion by giscus** [stoatchat/discussions#645] [Announcements]
# Refreshing Revolt for Web - Revolt

Bringing a new, leaner experience for web users.

https://revolt.chat/posts/refreshing-revolt-web

**Comment by DeclanChidlow** [comment]
Horray!

**Comment by ERROR-404-NULL-NOT-FOUND** [comment]
I wasn't stoked until I realized that it was public :trol:

**Comment by arslee07** [comment]
:trollface:

**Comment by Andre601** [comment]
Maintainer edit: this was due to confusion with the settings menu, the actual colour picker is the server list (I've added a note to clarify this).

---

Are you sure you tested this?
- Color picker: Not working
- Dark theme: Non existant (No way to select it... My eyes hurt now. Thanks)

Also, browser reports you use window.controller(s) for browser recognition. That stuff is outdated and shouldn't be used anymore... Like if my browser tells me that, I'm sure it's something to keep in mind.

Also, also, using Firefox on Windows 10.

**Comment by searinminecraft** [comment]
So far it looks really beautiful. Material Design 3 is by doubt one of my most favorite design languages of all time. Well it isn't really material as it does not use material icons or material elements like rippling buttons and the iconic material spinner,  but overall it's pretty decent. Keep up the good work!

Also common Revolt W

**Comment by FanMclaine** [comment]
yippie

**Comment by Kirito139** [comment]
Question: does material include any google tracking? Because that would suck. If not, then privacy for the W!

**Reply by Rexogamer** [reply]
Our implementation of Material for Revolt has been written in-house from the ground up. You'll be able to learn more about Revolt M3 Extensions on the Revolt Brand Hub in the near future.

**Reply by Kirito139** [reply]
Thanks for the quick response! So no google?

**Reply by Andre601** [reply]
I personally would assume so given it's "in-house", so made by them.

**Reply by Rexogamer** [reply]
Correct :3

**Comment by ghost** [comment]
poggers

**Comment by kayodde** [comment]
Why do the buttons to login on https://revolt.chat/app not work?

**Comment by Krojack** [comment]
Is there a problem with the site? I'm getting the following when going to it now. 

> Secure Connection Failed
> 
> An error occurred during a connection to revolt.chat. SSL received a record that exceeded the maximum permissible length.
> 
> Error code: SSL_ERROR_RX_RECORD_TOO_LONG
> 
>     The page you are trying to view cannot be shown because the authenticity of the received data could not be verified.
>     Please contact the website owners to inform them of this problem.
>

**Comment by coleishere2225** [comment]
What happened to revolt.chat/login? The webpage says it's not found. Is it out of maintenance or the login link has been changed? If it's out of maintenance right now does it explain why the login page is missing?

**Reply by Krojack** [reply]
Try this address... it's working for me

https://app.revolt.chat/login

**Reply by coleishere2225** [reply]
@Krojack That address that still uses the legacy client isn't the solution I expected and it's the same legacy client that has the message renderer for the timeline to scroll to the bottom on load is broken where the modern client has it fixed. 

I wish there was more third party web clients that has the scroll to bottom function fixed than just the official modern client of Revolt. I guess I have to live with using the legacy client for now until the Revolt Staff brings the modern client's login page back online who knows whenever in the future. 

I don't know If there is other chat software that has the scroll to bottom function working than just IRC.

Was looking forward on using the modern client to enjoy during December 24th to the 25th in the Christmas time  when I saw the codebase being updated for the Frontend repo before the Christmas days hit.
